Testscripts/Linux/validate-da.sh

```diff
@@ -220,20 +237,22 @@ function enable_disable_da(){
         fail_test "$da_log_dir/service.log.2 exist."
     fi
     
-    if [ ! -c "/dev/msda" ]; then 
-        fail_test "/dev/msda does not exist."
-    fi
-    
     # Check for "driver setup status=0" and "starting the dependency agent" in service.log.1 and service.log respectively
     
-    if ! driver_status=$(sed -n '/Driver setup status=//s/^[^=]*//p' $da_log_dir/service.log.1); then
-        fail_test "sed command failed while reading service.log.1 file"
+    if ! driver_status=$(sed -n '/Driver setup status=/s/^[^=]*//p' "$da_log_dir/service.log.1"); then
+        fail_test "sed command failed while reading $da_log_dir/service.log.1 file"
     fi
 
     case "$driver_status" in
         =0)
             # expected result
             ;;
```

Comment:
Also need to check for status 35. This indicates we've explicitly excluded the kernel before attempting to load the kernel module.  If this happens, we may choose to continue the test. The only difference will be that the sense of the `/dev/msda` assertion below is inverted. It should **not** exist and is a test failure if it does.

Reply:
Done